### ISSUE TITLE
Dependencies in cmake missing

### ISSUE THREAD

**Issue by fmauch** [mahkoh/spotblock#5]
Hi there,

I just compiled the package under Ubuntu and there seem to be dependencies missing in the CMake files.

cmake went down find, but when invoking make I got errors due to missing dependencies of pulse headers and systemd headers. Maybe they could be added to the cmake files as well.

Thanks for the work :)

**Comment by mahkoh** [comment]
You have to install the pulseaudio and systemd development packages.

**Comment by fmauch** [comment]
Yes, I do know that and did that, I just think it would be nice to check this in the CMake step already.
